c_cxx/QNN_EP/mobilenetv2_classification/CMakeLists.txt

```diff
@@ -0,0 +1,32 @@
+project(qnn_ep_sample C CXX)
+
+if(NOT CMAKE_BUILD_TYPE)
+  set(CMAKE_BUILD_TYPE Release)
+endif()
+
+cmake_minimum_required(VERSION 3.13)
+
+option(ONNXRUNTIME_ROOTDIR "onnxruntime root dir")
+
+set(CMAKE_CXX_STANDARD 14)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+
+include_directories(${ONNXRUNTIME_ROOTDIR}/include/onnxruntime/core/session/)
```

Comment:
i think this camke file is based on the one for the snpe example. i tried to clean up that example when creating https://github.com/microsoft/onnxruntime-inference-examples/blob/main/c_cxx/QNN_EP/QnnCpu/CMakeLists.txt
notice we assume that ort libs are in some root dir (rather than assume the user has a clone of ort repo)

Reply:
if we assume the user has clone of ort source tree at ONNXRUNTIME_ROOTDIR, let's document the assumption.

Reply:
Let me make the readme clear.

---
In reply to: [1203291967](https://github.com/microsoft/onnxruntime-inference-examples/pull/246#discussion_r1203291967) [](ancestors = 1203291967)